client/components/LearnerViewDirective.js

```diff
@@ -52,6 +54,15 @@ tie.directive('learnerView', [function() {
                     </span>
                   </p>
                 </div>
+                <div class="tie-feedback-syntax-error">
+                <a href class="tie-feedback-syntax-error-link", ng-click="toggleSyntaxErrorHint()", 
+                        ng-show="isSyntaxErrorLinkShown">
+                  {{isSyntaxErrorShown ? 'Hide error details' : 'Display error details'}}
+                </a>
+              </div>
```

Comment:
I was actually referring to the lines below &lt;div class="tie-feedback-syntax-error"&gt; not the lines above it. The alignment seems to be off now. Please review and make sure the tags match and line up.